src/bindings.cc

```diff
@@ -8,12 +8,17 @@
 #include "event.cc"
 #include "reporter.cc"
 
-NAN_METHOD(oboeInit) {
-
-    if (info.Length() < 1 || !info[0]->IsString() || (info.Length() > 1 && !info[1]->IsObject())) {
-        return Nan::ThrowError("oboeInit - invalid calling signature");
+//
+// Initialize oboe
+//
+Napi::Value oboeInit(const Napi::CallbackInfo& info) {
```

Comment:
Not all cases, but sometimes I return either an Object or Undefined or Null. Because JavaScript is not a strongly typed language it seemed to make sense to use Value. In the case of oboeInit() all it returns is null, and null is an object, so I could declare that it returns an object. An object is a Napi::Value, so that is OK either way. But it's just one more thing to keep track of if I change the code to allow it to return undefined for some path. And JavaScript doesn't benefit from the differentiation.

config.cc/getConfigSettings() can return undefined or an object. undefined is *not* an object (maybe not a great decision by the JavaScript designers but it is what it is). So any time a function can return more than one type it has to be a value.

I could play around with being more explicit on some of them but it didn't seem worth it because I can't do so consistently.